Userland/Utilities/dd.cpp

```diff
@@ -50,7 +53,13 @@ static void closing_statistics()
         warnln("{}+{} blocks out", total_blocks_out, partial_blocks_out);
     }
     if (status != Noxfer) {
-        warnln("{} bytes copied.", total_bytes_copied);
+        auto elapsed_time = timer.elapsed_time();
+        auto speed = total_bytes_copied / elapsed_time.to_seconds();
```

Comment:
I noticed a crash when trying out this change because the divisor `elapsed_time.to_seconds()` was rounded to zero :^)
```
152.277 [dd(43:43)]: CRASH: CPU #0 Divide error in userspace
152.277 [#0 dd(43:43)]: Exception code: 0000 (isr: 0000)
152.277 [#0 dd(43:43)]:     pc=0x0023:0x00000007fe77873a rflags=0x0000000000010246
152.277 [#0 dd(43:43)]:  stack=0x00000007669bd5c0
152.277 [#0 dd(43:43)]:    rax=0x0000000000000200 rbx=0x00000007669bd618 rcx=0x0000000000000000 rdx=0x0000000000000000
152.277 [#0 dd(43:43)]:    rbp=0x00000007669bd6b0 rsp=0x0000002011d6ff50 rsi=0x0000000000000000 rdi=0x00000007669bd5d8
152.277 [#0 dd(43:43)]:     r8=0x0000000000000003  r9=0x0000000000000001 r10=0x00000007669bd5d8 r11=0x0000000000000246
152.277 [#0 dd(43:43)]:    r12=0x00000007669bd600 r13=0x00000007669bd5f0 r14=0x000000160a40c300 r15=0x00000007fe7796b0
152.277 [#0 dd(43:43)]:    cr0=0x0000000080010013 cr2=0x00000007fe77b0df cr3=0x000000000e6a6000 cr4=0x0000000000340ee0
152.277 [#0 dd(43:43)]: 0x00000007fe77873a  (?)

152.277 [#0 dd(43:43)]: Kernel backtrace:
152.277 [#0 dd(43:43)]: Kernel + 0x0000000001121c69  Kernel::Process::crash(int, AK::Optional<Kernel::RegisterState const&>, bool) +0x359
152.277 [#0 dd(43:43)]: Kernel + 0x00000000001b1bb7  Kernel::handle_crash(Kernel::RegisterState const&, char const*, int, bool) +0x507
152.277 [#0 dd(43:43)]: Kernel + 0x0000000001705160  divide_error_asm_entry +0x30
152.298 [#0 Finalizer Task(5:5)]: Generating coredump for pid: 43
152.305 CrashDaemon(32:32): New coredump file: /tmp/coredump/dd_43_1684100891
152.483 CrashReporter(44:44): Started Thread "Background Thread"(45)
152.574 [#0 VFS Sync Task(4:4)]: Ext2FS: Flushed 4 blocks to disk
152.590 CrashReporter(44:45): Generating backtrace took 107 ms
152.590 CrashReporter(44:45): --- Backtrace for thread #0 (TID 43) ---
152.595 CrashReporter(44:45): 0x00000007fe77873a: [/bin/dd] closing_statistics() +0xaa (dd.cpp:57 => dd.cpp:49)
152.595 CrashReporter(44:45): 0x00000007fe77906b: [/bin/dd] serenity_main(Main::Arguments) +0x28b (dd.cpp:267)
152.598 CrashReporter(44:45): 0x00000007fe7783ea: [/bin/dd] main +0x14a (Main.cpp:43)
152.598 CrashReporter(44:45): 0x00000007fe778578: [/bin/dd] _entry +0x48 (crt0.cpp:48)
```
You can easily reproduce as follows:
```
dd if=/dev/zero of=./hello count=1
```